### PR TITLE
C program to create and sort a single Linked List.

### PR DESCRIPTION
**Pull request by omprakashchhetri**
Added a topic related to Linked List in C language ! #1

### CODE REVIEW

File: c/LinkedList.c [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+          {
+               case 1:
+               right=insert(right);
+               if(left==NULL)
+               {
+                   left=right;
+               }
+               break;
+               case 2:
+               sort(left);
```
Comment (anuragdevon) [inline c/LinkedList.c:33]: @omprakashchhetri after sorting, the menu-driven is getting terminatted, try to continue the program back to menu-driven.

File: c/LinkedList.c [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+               case 3:
+               exit(1);
+               default:
+               printf("invalid choice");
+               break;
+          }
+          printf("do you want to continuoe press 1: ");
```
Comment (anuragdevon) [inline c/LinkedList.c:40]: @omprakashchhetri fix the continue and comma typo.

**Review by anuragdevon — CHANGES_REQUESTED** [review]

File: c/LinkedList.c (hunk 1 above)
Comment (anuragdevon) [reply c/LinkedList.c:33]: possibly add a `break` so that it does not fall to `case 3` and exits.

**Review by anuragdevon — CHANGES_REQUESTED** [review]

File: c/LinkedList.c (hunk 1 above)
Comment (omprakashchhetri) [reply c/LinkedList.c:33]: Sure !

**Comment by omprakashchhetri** [comment]
I've made the changes can you please check ! @anuragdevon

**Review by anuragdevon — APPROVED** [review]

**Comment by omprakashchhetri** [comment]
Thank you for accepting my PR !

**Pull request merged**